keyboards/yoichiro/lunakey_macro/readme.md

```diff
@@ -0,0 +1,19 @@
+# Lunakey Macro
+
+![Lunakey Macro](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Yoichiro Tanaka](https://github.com/yourusername)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on hw to enter the bootloader?

Reply:
@drashna Copy that.